pkg/backend/aio.go

```diff
@@ -136,6 +136,11 @@ func (s *Server) UpdateAioController(_ context.Context, in *pb.UpdateAioControll
 // ListAioControllers lists Aio controllers
 func (s *Server) ListAioControllers(_ context.Context, in *pb.ListAioControllersRequest) (*pb.ListAioControllersResponse, error) {
 	log.Printf("ListAioControllers: Received from client: %v", in)
+	if in.PageSize < 0 {
+		err := status.Error(codes.InvalidArgument, "negative PageSize is not allowed")
```

Comment:
Would it make sense to keep common errors defined in one place (maybe in `server` pkg) to avoid the duplication?

Reply:
yep, good idea. I saw @artek-koltun did similar thing. thanks. opened #306